scene/animation/tween.cpp

```diff
@@ -844,3 +865,49 @@ MethodTweener::MethodTweener(const Callable &p_callback, const Variant &p_from,
 MethodTweener::MethodTweener() {
 	ERR_FAIL_MSG("MethodTweener can't be created directly. Use the tween_method() method in Tween.");
 }
+
+void SubtweenTweener::start() {
+	finished = false;
```

Comment:
```suggestion
	elapsed_time = 0;
	finished = false;
```
As in other Tweeners.